tests/runtime/conftest.py

```diff
@@ -93,7 +107,7 @@ def base_container_image(request):
 def runtime(temp_dir, box_class, run_as_openhands):
     runtime = _load_runtime(temp_dir, box_class, run_as_openhands)
     yield runtime
-    runtime.close()
+    runtime.close(rm_all_containers=False)
```

Comment:
I think this line, which was already here, is an actual bug: its "runtime.close" is after "yield" which means after every test.
But as far as I've seen, most if not all tests, which use the runtime method, have their own call to "runtime.close" already (which I prefer so containers get closed asap).
Please remove this line and lets see how the tests go.

Reply:
Updated